access-common/src/test/java/edu/unc/lib/dl/ui/util/SerializationUtilTest.java

```diff
@@ -15,44 +15,133 @@
  */
 package edu.unc.lib.dl.ui.util;
 
-import java.util.Arrays;
+import static java.util.Arrays.asList;
+import static java.util.Collections.singletonList;
+import static org.mockito.Mockito.when;
+
 import java.util.List;
+import java.util.Map;
 
+import org.codehaus.jackson.map.ObjectMapper;
+import org.codehaus.jackson.type.TypeReference;
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
 
+import edu.unc.lib.dl.search.solr.model.BriefObjectMetadata;
 import edu.unc.lib.dl.search.solr.model.BriefObjectMetadataBean;
+import edu.unc.lib.dl.search.solr.model.HierarchicalBrowseResultResponse;
+import edu.unc.lib.dl.search.solr.model.SearchResultResponse;
+import edu.unc.lib.dl.search.solr.util.SearchSettings;
+import edu.unc.lib.dl.search.solr.util.SolrSettings;
 
+/**
+ *
+ * @author bbpennel
+ *
+ */
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
why is this annotation needed for this test?

Reply:
I think I was trying it out after seeing it used somewhere else.  It makes it so you don't need to call `initMocks` in the setup method, which I guess is nice